src/docs/ChatContext.md

```diff
@@ -1,10 +1,16 @@
 The channel context provides the following properties:
 
-- **client** (the client connection)
+- **client** The Client connection
+- **theme** The current theme
+- **channel** The currently active channel
+- **closeMobileNav** The function to close mobile navigation
+- **mutes** An array of muted users
+- **navOpen** A boolean if navigation is open
+- **openMobileNav** A function to open mobile navigation
 - **setActiveChannel** A function to set the currently active channel. This is used in [ChannelList](#channellist) component to navigate between channels.
 
   **Params**
 
-  - `channel` Channel that needs to set to as active channel.
-
-- **channel** The currently active channel
+  - `newChannel` Channel that needs to be set as active Channel
+  - `watchers` water parameters: { limit?: number; offset?: number }
```

Comment:
water?

Reply:
whoops! Searched the whole project for water to be sure I didn't copy and paste this elsewhere too...